### ISSUE TITLE
WebAssembly support?

### ISSUE THREAD

**Issue by turbolent** [dylan-lang/opendylan#1418]
Given that OpenDylan now has a LLVM-based code generator, how much work would it be to add WebAssembly target?

**Comment by housel** [comment]
I'm pretty much waiting for https://github.com/WebAssembly/gc to progress further before attempting to pursue this.

**Comment by turbolent** [comment]
Right, the GC proposal would help. In the mean-time, could an implementation require a host-environment that provides memory management functions? The module would import and use those for allocating memory, the module would export its memory, and the host-environment would perform GC.

Are there places in the code base that I could look at?
